HelpSource/Classes/Pclump.schelp

```diff
@@ -0,0 +1,38 @@
+TITLE:: Pclump
+summary:: Clumps the pattern into groups of size n.
+related:: Classes/SequenceableCollection#-clump
+categories:: Streams-Patterns-Events>Patterns>List
+
+DESCRIPTION::
+Takes the source pattern into groupings whose size is given by integers in the pattern n.
+
+E.g. If the source pattern has 5 elements and you choose a clump size of 2, the new pattern will return two groupings of size 2, and a final grouping of size 1.
+
+CLASSMETHODS::
+
+METHOD:: new
+
+ARGUMENT:: n
+a pattern that returns an integer. The returned value will determine the size of the next clump.
```

Comment:
while i personally know that integer arguments to this parameter will be converted to a pattern under the hood, a newer user might now. so I would suggest writing "an integer, or a pattern that returns an integer."

Reply:
Yes that makes sense.